### ISSUE TITLE
Regression in 0.9.14: illegal option -- -

### ISSUE THREAD

**Issue by yurivict** [antoinemine/apron#98]
```
gmake[2]: Entering directory '/usr/ports/math/apron/work/apron-0.9.14/itv'
/usr/bin/install -d /usr/ports/math/apron/work/stage/usr/local/include /usr/ports/math/apron/work/stage/usr/local/lib
/usr/bin/install itv.h itv_fun.h itv_config.h itv_linexpr.h itv_linearize.h /usr/ports/math/apron/work/stage/usr/local/include
for i in libitvIl.a libitvIll.a libitvRl.a libitvRll.a libitvMPZ.a libitvMPQ.a libitvD.a libitvMPFR.a libitv.a libitvDl.a; do \
        if test -f $i; then /usr/bin/install --strip $i /usr/ports/math/apron/work/stage/usr/local/lib; fi; \
done
install: illegal option -- -
usage: install [-bCcpSsUv] [-f flags] [-g group] [-m mode] [-o owner]
               [-M log] [-D dest] [-h hash] [-T tags]
               [-B suffix] [-l linkflags] [-N dbdir]
               file1 file2
       install [-bCcpSsUv] [-f flags] [-g group] [-m mode] [-o owner]
               [-M log] [-D dest] [-h hash] [-T tags]
               [-B suffix] [-l linkflags] [-N dbdir]
               file1 ... fileN directory
       install -dU [-vU] [-g group] [-m mode] [-N dbdir] [-o owner]
               [-M log] [-D dest] [-h hash] [-T tags]
               directory ...
```

It looks like the build scripts became Linux-only since the release 0.9.14.
On BSDs install doesn't have such options.

FreeBSD 13.2

**Comment by antoinemine** [comment]
PR #97 might solve your problem as it uses `strip` directly to strip binaries instead of using the `--strip` option.

Could you please check whether this is the case, so that we can fix #97 for BSD before merging?

Until the merge, you can use the `-no-strip` configure option to disable `--strip`.

**Comment by antoinemine** [comment]
Could you check that PR #97 fixes the issue on FreeBSD?

**Comment by antoinemine** [comment]
Hi! Can you tell me if PR #97 fixed the issue for you?

**Comment by antoinemine** [comment]
Considered fixed by #97.
